src/js/views/maps/feature-tiles-layer.js

```diff
@@ -0,0 +1,236 @@
+/*jslint nomen: true */
+/*globals define */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+  var _ = require('lib/lodash');
+  var Backbone = require('backbone');
+  var L = require('lib/leaflet/leaflet.tilejson');
+  var Promise = require('lib/bluebird');
+
+  var api = require('api');
+  var util = require('util');
+
+  // We reuse the cartodb layer templates
+  var infoTemplate = require('text!templates/cartodb-info.html');
+  var tooltipTemplate = require('text!templates/cartodb-tooltip.html');
+  var template = require('text!templates/projects/layerControl.html');
+
+  // Render an infowindow for a selected map item
+  var ItemView = Backbone.View.extend({
```

Comment:
Do we use this in more than one place? 

Also, our recent trend has been to have one view per file.